### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by cereallarceny**
Don't merge this yet.

### CODE REVIEW

**Comment by cereallarceny** [comment]
@kevinahuber Steps from here:

1. Get SSL on api.openmined.org
2. Get the `blog` branch on the website repo deployed to staging
3. Test, test, test…
4. Fix anything that needs fixing
5. Clear out dummy content
6. Merge `blog` into `master`
7. Point the domain to the new server

**Comment by cereallarceny** [comment]
Currently, I believe we're also having a problem with SSR rendering metadata on the server for blog posts because it's pulled from the server.  I believe this is because the request is finishing AFTER the server is rendering the `<meta>` tags.  [Redux Tiles](https://github.com/Bloomca/redux-tiles#server-side-rendering) potentially offers a solution here.  @kevinahuber We should investigate this further with a proper spike.

**Comment by cereallarceny** [comment]
Testing and staging deployment is complete @kevinahuber.  Likewise, due to the removal of Serverless from the pipeline, we can update the current status of this PR to the following:

- [x] Set up Wordpress instance @cereallarceny
- [x] Connect to React app and create back-end with content @cereallarceny
- [x] Replace homepage content with Wordpress endpoint @cereallarceny
- [x] Build out blog (the brunt of the work in this PR) @cereallarceny
- [x] Get server-side rendering working with full endpoint SEO @cereallarceny
- [x] Get SSL on api.openmined.org @kevinahuber
- [x] Get the blog branch on the website repo deployed to staging @kevinahuber 
- [x] Test, test, test… @cereallarceny
- [x] Fix anything that needs fixing @cereallarceny
- [ ] Set up deployment pipeline for Node server @kevinahuber 
- [ ] Create first blog post @iamtrask 
- [ ] Clear out dummy content @cereallarceny
- [ ] Merge blog into master (i.e. this PR) @kevinahuber 
- [ ] Point the domain to the new server @kevinahuber 
- [ ] Tear down Serverless and remove repository @cereallarceny
- [ ] Setup deployment pipeline for Wordpress website @kevinahuber 
- [ ] Setup local Wordpress environment for open-source development @cereallarceny & @kevinahuber 

Long story short, this PR is ready and @iamtrask is able to create our first blog post!  Once this has been completed, we can remove all the posts, and go live.

**Comment by cereallarceny** [comment]
@kevinahuber I've readjusted the above items to fit a more reasonable timeline.  Would you mind tending to what's needed?
